attributes/default.rb

```diff
@@ -1,3 +1,5 @@
 default['ulimit']['pam_su_template_cookbook'] = nil
 default['ulimit']['users'] = Mash.new
 default['ulimit']['security_limits_directory'] = '/etc/security/limits.d'
+default['ulimit']['ulimit_overriding_sudo_file_name'] = 'sudo'
+default['ulimit'][''ulimit_overriding_sudo_file_cookbook''] = nil
```

Comment:
Came here to merge this in, and noticed this accidental quoting issue - mind fixing it up real quick?

Thanks!